core/store/src/columns.rs

```diff
@@ -255,6 +255,29 @@ pub enum DBCol {
     FlatStateMisc,
 }
 
+#[derive(PartialEq, Copy, Clone, Debug, Hash, Eq, strum::EnumIter)]
+pub enum DBKeyType {
+    Empty,
+    Literal,
```

Comment:
This needs more documentation.  Especially what Empty and Literal mean.

Reply:
I tried. Don't know if that helps.